components/ingest-service/backend/elastic/jobs.go

```diff
@@ -19,9 +19,63 @@ import (
 	"github.com/chef/automate/components/ingest-service/backend/elastic/mappings"
 )
 
+type nodeUUID struct {
+	EntityUUID string `json:"entity_uuid"`
+}
+
+// GetNodesToMarkMissing gets all the node that need updated to missing
+func (es *Backend) GetNodesToMarkMissing(ctx context.Context, threshold string) ([]string, error) {
```

Comment:
New elasticsearch function to collect all the nodes that need marked missing.

Reply:
i was gonna ask why a new function (why not just modify MarkNodesMissing to return the list of ids, but after digging in a bit I guess the BulkResponseItem/Request msg don't provide a way of doing that -- is that right?

Reply:
You are totally right, thank you for pointing it out. I wrote this with my original thought of how this function worked but did not change my tack when I saw that it was looping through the nodes. I have updated it locally and will push it up soon.